app/src/main/java/net/vonforst/evmap/fragment/MapFragment.kt

```diff
@@ -1001,6 +1001,8 @@ class MapFragment : Fragment(), OnMapReadyCallback, MenuProvider {
         markerManager = MarkerManager(context, map, this).apply {
             onChargerClick = {
                 vm.chargerSparse.value = it
+                binding.detailAppBar.toolbar.menu.findItem(R.id.menu_share).isVisible =
+                    it.url != null
```

Comment:
Let's rather do this in the observer (`vm.chargerSparse.observe` call in `setupObservers`). That is where the state of the favorite button in the toolbar is updated as well.